src/components/AktivBrukerTilgangLaster.tsx

```diff
@@ -17,7 +17,7 @@ const AktivBrukerTilgangLaster = ({
   children,
 }: AktivBrukerTilgangLasterProps): ReactElement => {
   const {
-    isLoading: henterTilgang,
+    isInitialLoading: henterTilgang,
```

Comment:
Endret til isInitialLoading for queries som kan være disabled (Ref: https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-4#disabled-queries https://tanstack.com/query/v4/docs/react/guides/disabling-queries#isinitialloading)